bin/environment.yml

```diff
@@ -0,0 +1,7 @@
+name: myenv
```

Comment:
nit: change this to something more meaningful

Reply:
name changed to `kv_pair_appender_env`